### ISSUE TITLE
~= and != look the same with ligatures enabled

### ISSUE THREAD

**Issue by orgads** [microsoft/cascadia-code#703]
### Cascadia family version

2111.01

### Cascadia family variant(s)

Cascadia Code (the version with ligatures)

### Font file format(s)

.ttf (static)

### Platform

Windows

### Other Software

Visual Studio Code

### What happened?

When font ligature `ss02` is enabled, `~=` and `!=` both become `≠`. This is very confusing.

**Comment by yvvt0379** [comment]
In Lua, `~=` is a symbol for checking whether a value **doesn't equal** another value.

**Comment by ExE-Boss** [comment]
Yeah, but when one accidentally writes `!=` instead of `~=` due to muscle memory, I’d expect it to produce a different ligature even with `ss02` (right half of `=!=`).
